src/components/dialog/areas/DialogFooterActions.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames'
+import { ReactElement, ReactNode } from 'react'
+import { useTheme } from '../../../framework'
+import { ClassNameProps } from '../../types'
+import { DialogSize } from '../Dialog'
+
+export enum DialogFooterActionsVariant {
```

Comment:
```suggestion
export enum DialogFooterActionsPosition {
```

Are these not more positions?

Reply:
I took the `Actions` component as reference, where it is defined like this:
```ts
export enum ActionSectionVariant {
  start = 'start',
  center = 'center',
  end = 'end',
}
```
What should we do? :)

Reply:
Sorry, but then I think we should also talk about the `Actions` itself, if they are right. For me the word variant is wrong here. I would not add something that does not fit well, only because the other component has somthing that also does not fit well.

It would be something completely different, when it would work great for `Actions`.